resources/assets/components/PostFilter/index.js

```diff
@@ -1,35 +1,60 @@
 import React from 'react';
 
+import StatusFilter from '../StatusFilter';
+import TagsFilter from '../TagsFilter';
+
 class PostFilter extends React.Component {
```

Comment:
I think we should rename this to be `FilterBar` or something more general so it isn't so tightly connected to the content it is filtering.

Reply:
ah yes I like that idea! I will change that now.